src/Stream.php

```diff
@@ -285,11 +285,11 @@ public function getMetadata($key = null)
         }
 
         $metadata = stream_get_meta_data($this->resource);
-        if (! array_key_exists($key, $metadata)) {
-            return null;
+        if (isset($metadata[$key])) {
```

Comment:
Use `??` here too

Reply:
If you target PHP 7.1+, you'll need to update the `composer.json`, rebase against develop, and change the branch this is against.